lib/ClangImporter/ClangImporter.cpp

```diff
@@ -1012,7 +1012,10 @@ ClangImporter::create(ASTContext &ctx,
 
   // Set up the file manager.
   {
-    if (!ctx.SearchPathOpts.VFSOverlayFiles.empty()) {
+    if (importerOpts.DebuggerSupport) {
+      // The debugger uses the virtual file system for reproducers.
+      instance.setVirtualFileSystem(ctx.SourceMgr.getFileSystem());
+    } else if (!ctx.SearchPathOpts.VFSOverlayFiles.empty()) {
```

Comment:
I'm starting to think it'd be more sensible to just always use it, so that we don't get different behavior in different configurations. @harlanhaskins, what do you think?

Reply:
Agreed, if we haven’t defined any VFS overlays, then the VFS is just the file system. Would be good to just use it all the time.

Reply:
FWIW, a few tests break if I move it down unconditionally. I'll do some digging and report back.